src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -29,11 +29,11 @@
   public static final int SEND_ERROR = 32603;
   public static final int JSON_ERROR = -100;
 
-  static final int NO_ERROR = 0;
-  static final int EWON_ERROR = 1;
-  static final int AUTH_ERROR = 2;
-  static final int WEB_ID_ERROR = 2;
-  static final int GENERIC_ERROR = 254;
+  public static final int NO_ERROR = 0;
+  public static final int EWON_ERROR = 1;
+  public static final int AUTH_ERROR = 2;
+  public static final int WEB_ID_ERROR = 2;
+  public static final int GENERIC_ERROR = 254;

```

Comment:
I suppose it doesn't fall under the scope of this commit, but it would be good to see Javadocs for these constants